x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/logic/detection_rules_client/detection_rules_client_interface.ts

```diff
@@ -23,6 +26,7 @@ export interface IDetectionRulesClient {
   deleteRule: (args: DeleteRuleArgs) => Promise<void>;
   upgradePrebuiltRule: (args: UpgradePrebuiltRuleArgs) => Promise<RuleResponse>;
   importRule: (args: ImportRuleArgs) => Promise<RuleResponse>;
+  importRules: (args: ImportRulesArgs) => Promise<Array<RuleResponse | RuleImportErrorObject>>;
```

Comment:
It wasn't a conscious decision, but was instead reusing the pattern already in place, where `ImportRuleResponse[]` was the return value.

The benefit is that the two code paths in the route can be handled identically, since their return values are the same.